modules/src/error_msg.py

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
This isn't a module. It's more of a helper class. Can be moved to modules/examples.py
Keep the base error in the module itself for now, and extract just the list of examples out. This should be used both for the error messages in src files, as well as in tests.